src/tools/cephfs/top/cephfs-top

```diff
@@ -249,13 +249,32 @@ class FSTopBase(object):
             self.stats_json = self.perf_stats_query()
             if fs_name:  # --dumpfs
                 if fs_name in fs_list:
-                    self.__build_clients(fs_name)
+                    self.__build_clients(fs_name, self.dump_json)
```

Comment:
We can do default initialization : `def __build_clients(self, fs, clients_json=self.dump_json):` instead of passing `self.dump_json` here